src/services/mentors.js

```diff
@@ -944,10 +945,7 @@ module.exports = class MentorsHelper {
 			return responses.successResponse({
 				statusCode: httpStatusCode.ok,
 				message: userDetails.message,
-				result: {
```

Comment:
@nevil-mathew why response is changed

Reply:
The response structure is same for 'extensionDetails', so removed the unnecessary mapping.

Reply:
@nevil-mathew where you are getting  extensionDetails.count